core/src/main/java/hudson/model/Queue.java

```diff
@@ -1854,7 +1853,7 @@ public Api getApi() {
      * transient Task, extend {@link TransientTask} marker interface.
      *
      * <p>
-     * Plugins are encouraged to extend from {@link AbstractQueueTask}
+     * Plugins are encouraged to extend from {@link Queue.Task}
      * instead of implementing this interface directly, to maintain
      * compatibility with future changes to this interface.
      *
```

Comment:
No you are missing the point of the original! Just delete this whole paragraph as it is now obsolete.